### PR TITLE
Refactor/update service status

### PR DESCRIPTION
**Pull request by facundoctes**
- [x]  Update displayed service state
- [x]  Update service state polling logic
- [x]  Update service card UI

### CODE REVIEW

File: src/views/service/BotServiceStatus.tsx [hunk 1]
```diff
@@ -16,11 +16,22 @@ const ServicePage: React.FC = () => {
...
+  );
+  const isStoping: boolean = useSelector((state: IAppState) =>
+    selectRequesting(state, [BotServiceActions.REQUEST_STOP_SERVICE])
+  );
 
   React.useEffect(() => {
     // Array of one virtual machine will be fetched
-    dispatch(getBotServiceAsync());
-  }, []);
+    if (!isStarting && !isStoping) {
```
Comment (JonatanMedinilla) [inline src/views/service/BotServiceStatus.tsx:28]: **nitpick(blocking)**: 
- We are enabling the polling only if the service is not starting or stopping, but what would happen if we have multiple services?
- We don't use setTimeOut in components. We have to use the useInterval hook.

**Review by JonatanMedinilla — CHANGES_REQUESTED** [review]
review comments

File: src/views/service/BotServiceStatus.tsx [hunk 2]
```diff
@@ -9,18 +9,22 @@ import * as BotServiceActions from '../../stores/service/actions';
...
   const { botServices } = useSelector((state: IAppState) => state.botServiceStatus);
+  const [isPollingEnabled, setIsPollingEnabled] = useState(false)
   const isRequesting: boolean = useSelector((state: IAppState) =>
     selectRequesting(state, [BotServiceActions.REQUEST_BOT_SERVICE])
   );
+ 
+  useInterval(useCallback(() => dispatch(getBotServiceAsync()), [dispatch, getBotServiceAsync]), isPollingEnabled ? 3000 : null);
+
+  useEffect(()=>{
+    setIsPollingEnabled(true)
```
Comment (sw-dcherubini) [inline src/views/service/BotServiceStatus.tsx:25]: **question**: I understand that we start the polling process as soon as we render this tab. But how is the polling process stopped? If it's not stopped, what happens if I switch to another tab and then switch back to this tab? Will I have multiple polling processes running?

Comment (facundoctes) [reply src/views/service/BotServiceStatus.tsx:25]: The useInterval hook calls clearInterval() when unmounting, so polling stops when you switching tabs

**Review by sw-dcherubini — APPROVED** [review]

**Pull request merged**